grpc/proto/v1/search_get.proto

```diff
@@ -244,6 +244,7 @@ message PropertiesResult {
   repeated ObjectProperties object_properties = 9 [deprecated = true];
   repeated ObjectArrayProperties object_array_properties = 10 [deprecated = true];
   Properties non_ref_props = 11;
+  bool ref_props_present = 12;
```

Comment:
I find the name a bit confusing, we want to distinguish between "no ref requested (and therefore empty list return)" and "ref requested, but nothing there". Can't think of anything better though 🤔